### PR TITLE
add help_text in courserun title and dates for syncing from edX studio course

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [x] Migrations
  - [x] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
NA

#### What's this PR do?
Add help_text in CourseRun model fields(title and dates) about syncing from edX studio course

#### How should this be manually tested?
Just visit the courserun page on admin

#### Screenshots (if appropriate)
Screenshot 2021-09-16 at 13 09 20

### CODE REVIEW

File: courses/migrations/0004_add_help_text_in_courserun_fields.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="courserun",
+            name="end_date",
+            field=models.DateTimeField(
+                blank=True,
+                db_index=True,
+                help_text="It will be fetched from edX studio course details via sync course run data",
```
Comment (briangrossman) [inline courses/migrations/0004_add_help_text_in_courserun_fields.py:19]:
```suggestion
                help_text="The last day the course is active. This value is synced automatically with edX studio.",
```

File: courses/migrations/0004_add_help_text_in_courserun_fields.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="courserun",
+            name="enrollment_end",
+            field=models.DateTimeField(
+                blank=True,
+                db_index=True,
+                help_text="It will be fetched from edX studio course details via sync course run data",
```
Comment (briangrossman) [inline courses/migrations/0004_add_help_text_in_courserun_fields.py:29]:
```suggestion
                help_text="The last day students can enroll. This value is synced automatically with edX studio.",
```

File: courses/migrations/0004_add_help_text_in_courserun_fields.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="courserun",
+            name="enrollment_start",
+            field=models.DateTimeField(
+                blank=True,
+                db_index=True,
+                help_text="It will be fetched from edX studio course details via sync course run data",
```
Comment (briangrossman) [inline courses/migrations/0004_add_help_text_in_courserun_fields.py:39]:
```suggestion
                help_text="The first day students can enroll. This value is synced automatically with edX studio.",
```

File: courses/migrations/0004_add_help_text_in_courserun_fields.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="courserun",
+            name="start_date",
+            field=models.DateTimeField(
+                blank=True,
+                db_index=True,
+                help_text="It will be fetched from edX studio course details via sync course run data",
```
Comment (briangrossman) [inline courses/migrations/0004_add_help_text_in_courserun_fields.py:49]:
```suggestion
                help_text="The day the course begins. This value is synced automatically with edX studio.",
```

File: courses/migrations/0004_add_help_text_in_courserun_fields.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="courserun",
+            name="title",
+            field=models.CharField(
+                help_text="It will be fetched from edX studio course details via sync course run data",
```
Comment (briangrossman) [inline courses/migrations/0004_add_help_text_in_courserun_fields.py:57]:
```suggestion
                help_text="The title of the course. This value is synced automatically with edX studio.",
```

**Review by briangrossman — COMMENTED** [review]
@umarmughal824 
I made some suggestions for the wording.

File: courses/constants.py [hunk 6]
```diff
@@ -31,3 +31,5 @@
 ENROLL_CHANGE_STATUS_CHOICES = list(
     zip(ALL_ENROLL_CHANGE_STATUSES, ALL_ENROLL_CHANGE_STATUSES)
 )
+
+SYNC_COURSE_RUN_STRING = "This value is synced automatically with edX studio."
```
Comment (gsidebo) [inline courses/constants.py:35]:
```suggestion
SYNCED_COURSE_RUN_FIELD_MSG = "This value is synced automatically with edX studio."
```

**Review by gsidebo — APPROVED** [review]
Looks great! Just one suggestion about renaming the constant. You can merge right away after making that change and tests pass 👍

**Pull request merged**